packages/web-components/src/slider/slider.styles.ts

```diff
@@ -0,0 +1,199 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusCircular,
+  borderRadiusMedium,
+  borderRadiusSmall,
+  colorBrandBackground,
+  colorCompoundBrandBackground,
+  colorCompoundBrandBackgroundHover,
+  colorCompoundBrandBackgroundPressed,
+  colorNeutralBackground1,
+  colorNeutralBackgroundDisabled,
+  colorNeutralForegroundDisabled,
+  colorNeutralStroke1,
+  colorNeutralStrokeAccessible,
+  colorNeutralStrokeDisabled,
+  colorStrokeFocus2,
+} from '../theme/design-tokens.js';
+
+/** Text styles
+ * @public
+ */
+export const styles = css`
+  ${display('inline-grid')} :host {
+    --thumb-size: 18px;
+    --thumb-padding: 3px;
+    --thumb-translate: calc(var(--thumb-size) * -0.5 + var(--track-width) / 2);
+    --track-overhang: -2px;
+    --track-width: 4px;
+    --fast-slider-height: calc(var(--thumb-size) * 10);
+    --slider-direction: 90deg;
+    align-items: center;
+    box-sizing: border-box;
+    outline: none;
+    cursor: pointer;
+    user-select: none;
+    border-radius: ${borderRadiusSmall};
+  }
+
+  :host([size='small']) {
+    --thumb-size: 14px;
+    --track-width: 2px;
+    --thumb-padding: 3px;
+  }
+  :host([orientation='horizontal']) {
+    touch-action: pan-y;
+    min-width: calc(var(--thumb-size) * 1px);
+    width: 100%;
+  }
+
+  :host([orientation='vertical']) {
+    --slider-direction: 0deg;
+    height: 160px;
+    min-height: var(--thumb-size);
+    touch-action: pan-x;
+    padding: 8px 0;
+  }
+
+  :host([disabled]:hover) {
+    cursor: initial;
+  }
+
+  :host(:focus-visible) {
+    outline-offset: 4px;
+    outline: 2px solid ${colorStrokeFocus2};
+  }
+
+  .thumb-cursor:focus {
+    outline: 0;
+  }
+
+  /* Thumb Container and Cursor */
+  .thumb-container {
+    position: absolute;
+    height: var(--thumb-size);
+    width: var(--thumb-size);
+    transition: all 0.2s ease;
+  }
+
+  :host([orientation='horizontal']) .thumb-container {
+    transform: translateX(calc(var(--thumb-size) * 0.5)) translateY(calc(var(--thumb-translate) * -1.5));
+  }
+
+  :host([orientation='horizontal'][size='small']) .thumb-container {
+    transform: translateX(calc(var(--thumb-size) * 0.5)) translateY(calc(var(--thumb-translate) * -1.35));
+  }
```

Comment:
Given this is the default i don't think you need to include the added selector for orientation here (do you?)